backend/server/src/main/java/ai/verta/modeldb/experimentRun/FutureExperimentRunServiceImpl.java

```diff
@@ -178,18 +93,7 @@ private InternalFuture<List<Void>> loggedDeleteExperimentRunEvents(List<String>
                     executor);
       }
 
-      var eventFuture =
-          projectIdFuture.thenCompose(
-              projectId ->
-                  addEvent(
-                      runId,
-                      Optional.empty(),
-                      projectId,
-                      DELETE_EXPERIMENT_RUN_EVENT_TYPE,
-                      Optional.empty(),
-                      Collections.emptyMap(),
-                      "experiment_run deleted successfully"),
-              executor);
+      InternalFuture<Void> eventFuture = projectIdFuture.thenCompose(projectId -> null, executor);
```

Comment:
returning a null future here in the `thenCompose` seems like it could break things

Reply:
this whole method should really go away and the callers removed

Reply:
this is actually the thing that's breaking tests